tests/particle_output_multiple_formats/statistics

```diff
@@ -0,0 +1,16 @@
+# 1: Time step number
+# 2: Time (seconds)
+# 3: Number of mesh cells
+# 4: Number of Stokes degrees of freedom
+# 5: Number of temperature degrees of freedom
+# 6: Number of degrees of freedom for all compositions
+# 7: Iterations for temperature solver
+# 8: Iterations for composition solver 1
+# 9: Iterations for Stokes solver
+# 10: Velocity iterations in Stokes preconditioner
+# 11: Schur complement iterations in Stokes preconditioner
+# 12: Time step size (seconds)
+# 13: Number of advected particles
+# 14: Particle file name(s)
```

Comment:
I think the format of the file endings is fine. About the length of the string: I can see that it is a bit annoying to read, but this file is also meant to be read by scripts and automatically processed, and for that it can have advantages to just list the full relative file. Adding a new column (that is the same for every timestep) seems not very useful to me, and might screw up existing scripts (I had to adjust a few scripts when we introduced the new solver iterations columns), but I could be convinced that adding the directory to the name of the 'Particle file name(s)' column would be an improvement. The new name of the column could be: "Particle file name(s) (relative to output-particle_output_multiple_formats/particles/)". But in general I have no strong feelings about it.
Whatever we decide about the particle file column should also be applied to the visualization file column to stay consistent.